app.js

```diff
@@ -42,18 +42,23 @@ app.use(cors())
 app.use(xss())
 app.use(rateLimiter())
```

Comment:
I think the "Too many requests" problem might be that you're calling the `rateLimiter` middleware twice, you should probably only do it once with the settings you need.